packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -27,12 +28,18 @@ type LaunchNodeOptions = {
   port?: string;
   args?: string[];
   useSystemFuelCore?: boolean;
+  useInMemoryDb?: boolean;
+  loggingEnabled?: boolean;
+  debugEnabled?: boolean;
+  basePath?: string;
+  poaInstant?: boolean;
```

Comment:
It is great to have these options as flags so they can be easily set like: `{ poaInstant: true }`. My concern is that many of them can be also set by using the `args` flag. 

My concern is that someone can use the `args` parameters to set options that are already being set with these other flags ( `useInMemoryDb`, `debugEnabled`, ...). 

What would happen if the flag `poaInstant` is set to `true` and within the `args` we have `['--poa-interval-period', '750ms']` ?

Not sure what would be the best approach here.

Just raising this concern to validate if it makes sense.

Reply:
True. If possible, it may be best to have only one way to set these.

Otherwise, we should at least validate conflicting settings.

Reply:
@Torres-ssf @arboleya great point. I decided against adding any extra validation because `fuel-core` handles this for us. If the user passes in any invalid options, they will get a readable errort that tells them that x flag is invalid and cannot be passed alongside y, z, etc. 

We should also note that this validation logic isn't exactly straightforward and changes frequently as new options are added/renamed. If we were to add extra validation in the TS SDK for this, we would be just duplicating this code from the client and we would always have to keep up with this validation logic. I don't think this is ideal.

Reply:
@arboleya I do agree that having only one way to set these things could be the way. In that case, I would favour only keeping the `args` property, and removing all the other params to maximize flexibility.

Reply:
It makes sense; having only `args` seems enough. 👍

Reply:
It seems like we hit this exact configuration conflict [here](https://github.com/FuelLabs/fuels-ts/actions/runs/7571055346/job/20617760404?pr=1602#step:9:372).

Reply:
@arboleya ah, must have crept in after we merged the latest changes from the rc branch. Should be fixed now